ydb/core/kqp/executer_actor/kqp_scheme_executer.cpp

```diff
@@ -120,6 +122,15 @@ class TKqpSchemeExecuter : public TActorBootstrapped<TKqpSchemeExecuter> {
         Become(&TKqpSchemeExecuter::ExecuteState);
     }
 
+    TString GetDatabaseName() const {
+        const auto appData = AppData();
+        if (appData && appData->AuthConfig.GetDomainLoginOnly()) {
```

Comment:
AppData() result cannot be nullptr if called from within an actor system (which it is).